pdns/dnsdist.hh

```diff
@@ -646,6 +646,16 @@ struct ServerPool
   NumberedVector<shared_ptr<DownstreamState>> servers;
   std::shared_ptr<DNSDistPacketCache> packetCache{nullptr};
   std::shared_ptr<ServerPolicy> policy{nullptr};
+
+  int countServersUp() {
+    int upFound = 0;
```

Comment:
The type of `upFound` and the function return value should be unsigned, so `size_t` would be a good choice.